package.json

```diff
@@ -9,7 +9,7 @@
     "index.js"
   ],
   "dependencies": {
-    "coffee-script": "~1.12.7",
+    "coffeescript": ">=2",
```

Comment:
Please use the `^2.4.1` style to refer to version 2. It shouldn't automatically pull in version 3 if a coffee-script version 3 is ever released.